openstack/manila/alerts/openstack-manila.alerts

```diff
@@ -71,6 +71,33 @@ groups:
       description: Manila Shares taking more than 2 minutes to resize in {{ $labels.manila_host }}
       summary: Manila Shares taking more than 2 minutes to resize
 
+  - alert: OpenstackManilaShareServersStuckCreate
+    expr: max(openstack_manila_share_servers_stuck_max_duration_gauge{status="creating"})
+      BY (manila_host) > 120
+    for: 5m
```

Comment:
Can you please increase to 10 or 15 mins?

Reply:
fixed, 15 mins